lib/src/plaform/linux.dart

```diff
@@ -14,10 +14,10 @@ int system(String command) {
   final systemP = dylib.lookupFunction<SystemC, SystemDart>('system');
 
   // Allocate a pointer to a Utf8 array containing our command.
-  final cmdP = Utf8.toUtf8(command);
+  final cmdP = command.toNativeUtf8();
 
   // Invoke the command, and free the pointer.
-  int result = systemP(cmdP);
+  final result = systemP(cmdP);
 //  cmdP.free();
 
```

Comment:
`calloc.free(cmdP);` otherwise you leak calloc'ed memory.

Same for the other files.

Reply:
done